### PR TITLE
Improved readability, imports, performance, ping exploit

### PR DESCRIPTION
**Pull request by DevilJamJar**
Improved indentation for readability in files, shuffled imports in a logical and readable manner and removed useless/unused imports, changed performance like `== None`, removed bare except in return for a specified Exception and a small change to the Pokedex command preventing ping exploittation

### CODE REVIEW

**Review by kyle-he — CHANGES_REQUESTED** [review]
Discord converts text surrounded by `` to `codeblocks`

File: helpers/constants.py [hunk 1]
```diff
@@ -63,10 +63,10 @@
 
 REWARD_WEIGHTS = {
```
Comment (kyle-he) [inline helpers/constants.py:64]: put back the original formatting pls

**Comment by DevilJamJar** [comment]
Yes, the issue was that a command such as " p!pokedex i`@everyone` " would escape the codeblock and ping the role anyway. By adding a space after the @, this technique will no longer work.

**Comment by oliver-ni** [comment]
I don't quite like the indentation changes, and your formatting messed up some of the parts that were manually formatted as well. For the @ everyone issue, replacing `@` with `@` isn't a particularly good way to solve it either—allowed_mentions exists and is for this purpose.

There are a couple good things but I can't cherrypick them out, so I'll be closing this—a couple of the issues I just addressed in 49c7dba673e331a68ab30089d6b30db6365c8947

**Pull request closed without merging**